src/Curation/Curation.router.ts

```diff
@@ -341,13 +341,9 @@ export class CurationRouter extends Router {
       )
     }
 
-    return curationService.getModel().update(
-      {
-        ...curation,
-        status: curationJSON.status,
-        updated_at: this.getISODate(),
-      },
-      { id: curation.id }
+    return await curationService.updateStatusAndReturnById(
```

Comment:
Same as above, we need the updated row, so I created this method to update the `status` and `updated_at` fields and return the affected row